docs: add config.rsbuild.source (#449)
web-infra-dev/rslib: website/docs/en/config/rsbuild.mdx

import { RsbuildDocBadge } from '@components/RsbuildDocBadge';
import { Overview } from 'rspress/theme';

# Rsbuild Configurations

> Rslib inherits its configuration from Rsbuild, so you can also configure the <RsbuildDocBadge path="/config" text="config" /> options from Rsbuild. Here are some common configuration items and how to use them in Rslib.

Reviewer: ```suggestion
Rslib inherits its configuration from Rsbuild, so you can also configure the <RsbuildDocBadge path="/config" text="config" /> options from Rsbuild. This chapter introduces some common configuration items and explains how to use them in Rslib.
```
